src/ApiPort/Proxy/ProxyProvider.cs

```diff
@@ -243,7 +243,7 @@ private static bool IsSystemProxySet(Uri uri)
                 if (proxyUri != null)
                 {
                     var proxyAddress = new Uri(proxyUri.AbsoluteUri);
-                    if (string.Equals(proxyAddress.AbsoluteUri, uri.AbsoluteUri))
+                    if (string.Equals(proxyAddress.AbsoluteUri, uri.AbsoluteUri, StringComparison.Ordinal))
```

Comment:
Hmm... should it be `StringComparison.OrdinalIgnoreCase`?
https://stackoverflow.com/questions/15641694/are-uris-case-insensitive

Reply:
Good point. I wasn't sure  - I've swapped it to ignore case